src/js/tether.js

```diff
@@ -730,7 +730,18 @@ class TetherClass {
       transcribe({top: true, left: true}, pos.page);
     }
 
-    if(!moved && this.element.parentNode.tagName !== 'BODY') {
+    let offsetParentIsBody = true;
+    let currentNode = this.element.parentNode;
+    while (currentNode && currentNode.tagName !== 'BODY') {
```

Comment:
You can move the `if (!moved)` check above this code to prevent it from having to be executed if it's already clear we're not gonna move the element again.